dev-libs/libxmlb/libxmlb-0.3.22.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="A library to help create and query binary XML blobs"
+DESCRIPTION="libxmlb is a small helper library that is used to create and \
+query the binary XML-like blobs that are used by AppStream."
+HOMEPAGE="https://github.com/hughsie/libxmlb"
+COPYRIGHT="Richard Hughes"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+
+SOURCE_URI="https://github.com/hughsie/libxmlb/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="103684ed37a45d0aed8f95e97294ed26945b5aeebf44734f3994081eecebb11c"
+SOURCE_FILENAME="libxmlb-$portVersion.tar.gz"
+SOURCE_DIR="libxmlb-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="2.0.0"
```

Comment:
Little comment on this, previous you had "2" here, looking at the package content you could see that the actual library is `libxmlb.so.2.0.0` with a link to `libxmlb.so.2 `and `libxmlb.so` (last one moved to _devel package), so l`ibVersionCompat = 2.0.0 compat >= 2`
You could check the SONAME also with `readelf -S libxmlb.so.2.0.0`

Reply:
Thank you, I'm not aware exactly how the library resolution works, but I think I'm understanding a bit more now.